app/(pages)/page.tsx

```diff
@@ -15,6 +15,8 @@ import { GA_TRACKING_ID } from '../settings';
 const Home = () => {
   return (
     <>
+    <Script src="../path/to/flowbite/dist/datepicker.js"></Script>
```

Comment:
Why is this import here?
We use DaisyUI as the UI Library. Please start using that. If DaisyUI doesn't have a datepicker then use a generic react datepicker library.